README.md

```diff
@@ -1,4 +1,8 @@
 # Kubeflow Operators
+[![CharmHub Badge](https://charmhub.io/kubeflow/badge.svg)](https://charmhub.io/kubeflow)
+[![Release](https://github.com/canonical/bundle-kubeflow/actions/workflows/release-bundle-to-charmhub.yaml/badge.svg)](https://github.com/canonical/bundle-kubeflow/actions/workflows/release-bundle-to-charmhub.yaml)
```

Comment:
what about `[run-tests-and-publish-bundle.yaml](https://github.com/canonical/bundle-kubeflow/blob/main/.github/workflows/run-tests-and-publish-bundle.yaml)`?

Reply:
I'd propose we would omit this one, since it's a workflow dispatch one that is used either way from the testing badge I added